packages/widgets/src/components/ExchangeAccount/types.ts

```diff
@@ -0,0 +1,11 @@
+import { AccountBalanceProps } from '@components/AccountBalance';
+
+export interface ExchangeAccountProps {
+  platform: string | JSX.Element;
+  title: string;
+  status: 'Active' | 'NotAvailable';
+  onClick?: () => void;
+  deleteOnClick: () => void;
```

Comment:
I would call it `onDelete` or `onRemove`

Reply:
Done.